Runtime/ScreenshotFilenameStrategies/FrameCountBasedStrategy.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2023 Koji Hasegawa.
+// This software is released under the MIT License.
+
+using System;
+using System.Runtime.CompilerServices;
+using UnityEngine;
+
+namespace TestHelper.Monkey.ScreenshotFilenameStrategies
+{
+    /// <summary>
+    ///     Frame count based screenshot file path strategy.
+    /// </summary>
+    public static class FrameCountBasedStrategy
```

Comment:
I don't want to include `FrameCountBasedStrategy` in the built-ins.
I want to implement only one of the built-in.